convert2rhel/systeminfo.py

```diff
@@ -403,10 +414,19 @@ def corresponds_to_rhel_eus_release(self):
         For example if we detect CentOS Linux 8.4, the corresponding RHEL 8.4 is an EUS release, but if we detect
         CentOS Linux 8.5, the corresponding RHEL 8.5 is not an EUS release.
 
-        :return: Whether or not the current system has a EUS correspondent in RHEL.
+        :return: Whether or not the current system has an EUS correspondent in RHEL.
         :rtype: bool
         """
-        return "%s.%s" % (self.version.major, self.version.minor) in EUS_MINOR_VERSIONS
+        current_version = "%s.%s" % (self.version.major, self.version.minor) in EUS_MINOR_VERSIONS
```

Comment:
This current_version being a boolean is kinda of wrong here... I think that we can remove the `in EUS_MINOR_VERSIONS` and let the variable behave just like a string that has the composition of the versions.

Reply:
Ok sounds good

Reply:
Well the `in EUS_MINOR_VERSIONS` is so we know that we are dealing with systems that are listed in that dictionary

Reply:
You won't get rid of the `EUS_MINOR_VERSIONS` check. You will only reorganize it a bit so the `current_version` variable actually represents a version instead of a flag. 

```suggestion
        current_version = "%s.%s" % (self.version.major, self.version.minor)
```

Later in the code, 

```python
        if current_version == "8.6":
            return current_version
```

and, 

```python
        if tool_opts.eus and current_version in EUS_MINOR_VERSION:
            self.logger.info("EUS argument detected, automatically evaluating system as EUS")
            return current_version
```